### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PY VII Instrument Cal CSVs [DT-259]](https://trello.com/c/STAbQG5Q/259-py-vii-instrument-cal-csvs-dt-259)

File: calibration/TURBDX/CGINS-TURBDX-00993__20240826.csv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+993,CC_dark_counts_turbd,50,source_file FLORT-D_BBFL2W_SN_993_Calibration_Files_NTU_2024-08-26.zip > BBFL2W-993.dev
+993,CC_scale_factor_turbd,6.239E-02,
+993,CC_cal_range,"[0,250]",
```
Comment (cooleyky) [inline calibration/TURBDX/CGINS-TURBDX-00993__20240826.csv:4]: looks good

File: calibration/TURBDX/CGINS-TURBDX-01102__20240925.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+1102,CC_dark_counts_turbd,40,source_file FLORT-D_BBFL2W_SN_1102_NTU_Characterization_2024-09-25.pdf
+1102,CC_scale_factor_turbd,0.0007,
+1102,CC_cal_range,"[0,3]",
```
Comment (cooleyky) [inline calibration/TURBDX/CGINS-TURBDX-01102__20240925.csv:4]: looks good

File: calibration/TURBDX/CGINS-TURBDX-01212__20240730.csv [hunk 3]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+1212,CC_dark_counts_turbd,47,source_file FLORT-D_BBFL2W_SN_1212_NTU_Characterization_2024-07-30.pdf
+1212,CC_scale_factor_turbd,0.0007,
+1212,CC_cal_range,"[0,3]",
```
Comment (cooleyky) [inline calibration/TURBDX/CGINS-TURBDX-01212__20240730.csv:4]: looks good

File: calibration/TURBDX/CGINS-TURBDX-01107__20240731.csv [hunk 4]
```diff
@@ -0,0 +1,4 @@
+serial,name,value,notes
+1107,CC_dark_counts_turbd,49,source_file FLORT-D_BBFL2W_SN_1107_NTU_Characterization_2024-07-31.pdf
+1107,CC_scale_factor_turbd,0.0009,
+1107,CC_cal_range,"[0,3]",
```
Comment (cooleyky) [inline calibration/TURBDX/CGINS-TURBDX-01107__20240731.csv:4]: looks good if we trust the characterization sheet

**Review by cooleyky — COMMENTED** [review]
Posting comments for review in process. All reviewed CSVs look good.

File: calibration/OPTAAD/CGINS-OPTAAD-00241__20240923__CC_taarray.ext [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+-0.007300,-0.005364,-0.004802,-0.004624,-0.003701,-0.003671,-0.004000,-0.003896,-0.003423,-0.003366,-0.003306,-0.002777,-0.002630,-0.002471,-0.001980,-0.002059,-0.002009,-0.001843,-0.001561,-0.001231,-0.000947,-0.000889,-0.000845,-0.000540,-0.000285,0.000000,-0.000076,0.000279,0.000426,0.000329,0.000620,0.000504,0.000512,0.000425,0.000462,-0.000203
+-0.007127,-0.005251,-0.004698,-0.004488,-0.003495,-0.003573,-0.003954,-0.003874,-0.003288,-0.003413,-0.003236,-0.002852,-0.002618,-0.002345,-0.001965,-0.001996,-0.002027,-0.001822,-0.001644,-0.001271,-0.000998,-0.000928,-0.000915,-0.000590,-0.000383,0.000000,-0.000099,0.000121,0.000372,0.000352,0.000451,0.000458,0.000466,0.000437,0.000484,0.000203
+-0.007337,-0.005777,-0.005207,-0.004889,-0.003906,-0.003854,-0.004361,-0.004157,-0.003706,-0.003666,-0.003375,-0.003066,-0.002765,-0.002563,-0.002079,-0.002207,-0.002178,-0.001939,-0.001640,-0.001306,-0.001000,-0.000858,-0.000910,-0.000513,-0.000189,0.000000,-0.000151,0.000219,0.000458,0.000452,0.000627,0.000496,0.000545,0.000449,0.000603,-0.000398
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00241__20240923__CC_taarray.ext:89]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00241__20240923__CC_tcarray.ext [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+-0.002637,-0.000086,0.000250,0.000077,0.000361,0.000231,0.000227,0.000279,0.000457,0.000120,0.000138,0.000558,0.000051,0.000379,0.000399,0.000198,0.000422,0.000789,0.000456,0.000253,0.000398,0.000343,0.000084,0.000315,0.000290,0.000000,0.000068,0.000121,-0.000252,-0.000325,-0.000517,-0.000811,-0.001000,-0.001292,-0.001322,-0.002202
+-0.002723,-0.000145,-0.000210,0.000030,-0.000207,-0.000296,-0.000389,-0.000114,-0.000318,-0.000178,-0.000066,0.000143,0.000315,0.000298,0.000133,0.000057,0.000131,0.000384,0.000555,0.000386,0.000531,0.000440,0.000303,0.000390,0.000504,0.000000,-0.000032,-0.000156,-0.000144,-0.000232,-0.000496,-0.000732,-0.000879,-0.000814,-0.001254,-0.001072
+-0.003408,-0.000498,-0.000261,-0.000369,-0.000149,-0.000015,-0.000169,0.000038,0.000115,0.000036,0.000051,0.000070,0.000252,0.000415,0.000286,0.000507,0.000442,0.000563,0.000337,0.000221,0.000263,0.000295,-0.000195,-0.000227,-0.000061,0.000000,0.000013,-0.000240,-0.000644,-0.000541,-0.000427,-0.000961,-0.001017,-0.001446,-0.001865,-0.002675
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00241__20240923__CC_tcarray.ext:89]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20241015.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+ACS-245,CC_tbins,"[0.616314, 1.363883, 2.423846, 3.438437, 4.466522, 5.476667, 6.472500, 7.473200, 8.495600, 9.501667, 10.503333, 11.506250, 12.517500, 13.481250, 14.489565, 15.475417, 16.490833, 17.494000, 18.495417, 19.497600, 20.476923, 21.499130, 22.510800, 23.501154, 24.489630, 25.513333, 26.498261, 27.476154, 28.493846, 29.510800, 30.510833, 31.519231, 32.503333, 33.484000, 34.510909]",
+ACS-245,CC_taarray,SheetRef:CC_taarray,
+ACS-245,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00245__20241015.csv:9]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20241015__CC_taarray.ext [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+-0.003897,-0.003669,-0.003593,-0.003026,-0.002757,-0.002924,-0.002686,-0.002348,-0.002316,-0.002276,-0.002246,-0.001848,-0.001327,-0.000935,-0.000830,-0.000924,-0.000785,-0.000916,-0.000547,-0.000511,-0.000381,-0.000399,-0.000250,-0.000197,-0.000068,0.000000,-0.000032,-0.000206,-0.000121,0.000062,0.000251,0.000277,0.000499,0.000701,0.000880
+-0.003368,-0.003253,-0.003256,-0.002694,-0.002616,-0.002610,-0.002420,-0.002225,-0.002159,-0.002198,-0.002170,-0.001759,-0.001251,-0.000708,-0.000720,-0.000854,-0.000787,-0.000807,-0.000461,-0.000428,-0.000210,-0.000201,-0.000081,-0.000175,-0.000002,0.000000,0.000084,0.000040,0.000137,0.000336,0.000383,0.000474,0.000626,0.000872,0.000999
+-0.004042,-0.003983,-0.004005,-0.003366,-0.003162,-0.003271,-0.003018,-0.002733,-0.002704,-0.002547,-0.002728,-0.002041,-0.001568,-0.001029,-0.000957,-0.001032,-0.000953,-0.001123,-0.000711,-0.000607,-0.000436,-0.000420,-0.000264,-0.000205,-0.000086,0.000000,-0.000004,-0.000146,-0.000068,0.000125,0.000363,0.000359,0.000452,0.000705,0.000968
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00245__20241015__CC_taarray.ext:90]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20241015__CC_tcarray.ext [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+0.017536,0.014260,0.012586,0.011373,0.010661,0.009858,0.008952,0.008243,0.007697,0.007142,0.006607,0.005852,0.005442,0.004934,0.004379,0.004011,0.003257,0.002962,0.002384,0.002203,0.001706,0.001109,0.000941,0.000592,0.000410,0.000000,-0.000364,-0.000506,-0.001042,-0.000960,-0.001346,-0.001643,-0.001556,-0.001937,-0.001752
+0.017772,0.014340,0.012922,0.011637,0.010854,0.010038,0.009155,0.008454,0.007903,0.007534,0.006909,0.006005,0.005563,0.005043,0.004497,0.003954,0.003511,0.003328,0.002789,0.002414,0.001885,0.001281,0.001096,0.000796,0.000430,0.000000,-0.000296,-0.000438,-0.000789,-0.000762,-0.001257,-0.001388,-0.001427,-0.001657,-0.001542
+0.014838,0.013177,0.011821,0.010962,0.010356,0.009325,0.008634,0.007953,0.007482,0.006963,0.006380,0.005883,0.005417,0.005015,0.004496,0.004110,0.003498,0.003082,0.002586,0.002441,0.001623,0.001348,0.001143,0.000487,0.000379,0.000000,-0.000699,-0.000837,-0.001365,-0.001482,-0.001461,-0.002035,-0.002054,-0.002322,-0.002308
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00245__20241015__CC_tcarray.ext:90]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240916.csv [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+ACS-254,CC_tbins,"[0.424050, 1.348201, 2.444898, 3.470714, 4.460952, 5.487778, 6.535714, 7.478800, 8.505600, 9.505652, 10.512800, 11.507826, 12.481667, 13.504000, 14.495217, 15.490000, 16.500800, 17.498333, 18.502500, 19.496800, 20.492400, 21.512000, 22.497083, 23.492308, 24.480870, 25.490769, 26.490000, 27.468846, 28.473200, 29.485600, 30.484000, 31.486667, 32.500769, 33.494706, 34.495588]",
+ACS-254,CC_taarray,SheetRef:CC_taarray,
+ACS-254,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240916.csv:9]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240916__CC_taarray.ext [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+-0.006304,-0.004717,-0.004081,-0.003561,-0.003347,-0.003265,-0.003052,-0.002875,-0.003024,-0.002856,-0.002699,-0.002594,-0.002481,-0.001859,-0.001727,-0.001292,-0.000852,-0.000811,-0.000518,-0.000574,-0.000391,-0.000253,-0.000173,-0.000257,-0.000156,0.000000,-0.000005,-0.000182,-0.000011,-0.000389,-0.000316,-0.000330,-0.000447,-0.000880,-0.001060
+-0.006577,-0.004896,-0.004151,-0.003708,-0.003454,-0.003320,-0.003335,-0.003090,-0.002976,-0.002976,-0.002636,-0.002526,-0.002408,-0.001709,-0.001724,-0.001220,-0.000790,-0.000519,-0.000372,-0.000468,-0.000305,-0.000112,-0.000128,-0.000272,-0.000038,0.000000,0.000074,-0.000131,-0.000077,-0.000300,-0.000218,-0.000313,-0.000453,-0.000926,-0.000963
+-0.006755,-0.005128,-0.004419,-0.003959,-0.003615,-0.003494,-0.003399,-0.003270,-0.003405,-0.003265,-0.003065,-0.002974,-0.002826,-0.001932,-0.002167,-0.001497,-0.000967,-0.000812,-0.000705,-0.000645,-0.000507,-0.000262,-0.000198,-0.000343,-0.000191,0.000000,-0.000187,-0.000176,-0.000103,-0.000409,-0.000379,-0.000395,-0.000455,-0.001068,-0.001026
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240916__CC_taarray.ext:87]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240916__CC_tcarray.ext [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+-0.000660,0.000601,0.000214,-0.000405,-0.000723,-0.000881,-0.000839,-0.000215,0.000018,-0.000003,0.000179,0.000198,0.000321,0.000135,0.000498,0.000428,0.000118,0.000161,0.000531,0.000147,0.000133,0.000276,0.000220,0.000108,-0.000029,0.000000,0.000023,0.000013,0.000085,0.000018,0.000107,0.000101,0.000318,0.000463,0.000641
+0.000041,0.000434,-0.000332,-0.000914,-0.001006,-0.001055,-0.001012,-0.000771,-0.000747,-0.000674,-0.000533,-0.000349,-0.000558,-0.000206,-0.000271,-0.000227,-0.000335,-0.000097,0.000013,-0.000003,0.000149,-0.000065,-0.000039,0.000051,-0.000091,0.000000,-0.000075,0.000057,-0.000111,0.000151,0.000040,-0.000026,0.000557,0.000592,0.000573
+-0.000433,0.000584,-0.000069,-0.000228,-0.000618,-0.000692,-0.000515,-0.000269,0.000067,-0.000250,-0.000217,-0.000006,-0.000219,-0.000056,-0.000107,-0.000126,-0.000198,-0.000244,0.000042,-0.000154,-0.000022,-0.000133,-0.000165,-0.000284,-0.000107,0.000000,-0.000008,-0.000045,-0.000163,-0.000167,0.000241,0.000200,0.000382,0.000587,0.000709
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240916__CC_tcarray.ext:87]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20240926.csv [hunk 13]
```diff
@@ -0,0 +1,9 @@
...
+ACS-256,CC_tbins,"[0.446700, 1.473176, 2.457551, 3.468571, 4.473636, 5.515789, 6.505185, 7.495833, 8.499130, 9.458333, 10.470833, 11.454167, 12.482000, 13.499167, 14.498333, 15.486957, 16.488846, 17.495833, 18.492500, 19.513200, 20.509583, 21.472000, 22.482083, 23.493200, 24.504800, 25.495385, 26.490385, 27.497200, 28.503200, 29.484167, 30.482308, 31.494000, 32.501600, 33.514138, 34.487647]",
+ACS-256,CC_taarray,SheetRef:CC_taarray,
+ACS-256,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20240926.csv:9]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20240926__CC_taarray.ext [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+-0.000417,0.000338,0.000397,0.000194,0.000038,-0.000154,-0.000231,-0.000216,0.000046,0.000306,0.000309,0.000153,0.000075,-0.000072,0.000002,0.000019,-0.000020,-0.000144,-0.000073,-0.000103,-0.000337,-0.000189,-0.000248,-0.000341,-0.000159,0.000000,-0.000076,0.000011,0.000004,0.000100,-0.000160,0.000035,-0.000049,-0.000043,-0.000194
+-0.000619,0.000120,0.000163,-0.000014,-0.000134,-0.000399,-0.000456,-0.000449,-0.000021,0.000099,0.000194,0.000088,-0.000050,-0.000188,-0.000177,-0.000108,-0.000120,-0.000245,-0.000052,-0.000166,-0.000304,-0.000265,-0.000337,-0.000270,-0.000068,0.000000,0.000010,0.000129,0.000089,0.000104,-0.000177,0.000114,0.000111,0.000026,-0.000067
+-0.000781,-0.000038,0.000059,-0.000118,-0.000253,-0.000455,-0.000597,-0.000544,-0.000222,0.000003,0.000001,-0.000087,-0.000242,-0.000455,-0.000331,-0.000295,-0.000242,-0.000377,-0.000243,-0.000360,-0.000495,-0.000412,-0.000469,-0.000392,-0.000170,0.000000,-0.000160,-0.000025,0.000087,0.000013,-0.000280,0.000082,0.000031,-0.000004,-0.000173
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20240926__CC_taarray.ext:85]: spot checked, looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20240926__CC_tcarray.ext [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+0.016660,0.012986,0.011353,0.009758,0.008218,0.007033,0.006080,0.006492,0.005762,0.005151,0.004275,0.003929,0.003622,0.002928,0.002288,0.001922,0.001880,0.001264,0.001259,0.000994,0.000593,0.000528,0.000212,0.000121,-0.000236,0.000000,-0.000310,-0.000362,-0.000744,-0.001081,-0.001142,-0.001662,-0.001907,-0.002292,-0.002778
+0.016030,0.012697,0.010932,0.009534,0.008148,0.006895,0.006157,0.006147,0.005797,0.005005,0.004433,0.003764,0.003412,0.003037,0.002535,0.002065,0.001534,0.001197,0.000970,0.000817,0.000470,0.000464,0.000525,0.000481,0.000150,0.000000,-0.000091,-0.000195,-0.000268,-0.000556,-0.000723,-0.001036,-0.001333,-0.001933,-0.002146
+0.016389,0.012515,0.010690,0.009438,0.008012,0.006689,0.006360,0.006220,0.005684,0.004931,0.004083,0.003546,0.003101,0.002731,0.002048,0.001766,0.001448,0.001161,0.001092,0.000587,0.000419,0.000470,0.000247,0.000072,-0.000089,0.000000,-0.000097,-0.000163,-0.000576,-0.000880,-0.000848,-0.001386,-0.001531,-0.002058,-0.002469
```
Comment (cooleyky) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20240926__CC_tcarray.ext:85]: spot checked, looks good

**Review by cooleyky — APPROVED** [review]
looks good

File: calibration/TURBDX/CGINS-TURBDX-01319__20240924.csv
Comment (reedan88) [inline calibration/TURBDX/CGINS-TURBDX-01319__20240924.csv:1]: Looks good.

File: calibration/TURBDX/CGINS-TURBDX-01219__20240828.csv
Comment (reedan88) [inline calibration/TURBDX/CGINS-TURBDX-01219__20240828.csv:1]: Looks good.

File: calibration/TURBDX/CGINS-TURBDX-01107__20240731.csv
Comment (reedan88) [inline calibration/TURBDX/CGINS-TURBDX-01107__20240731.csv:1]: Looks good

File: calibration/TURBDX/CGINS-TURBDX-01102__20240925.csv
Comment (reedan88) [inline calibration/TURBDX/CGINS-TURBDX-01102__20240925.csv:1]: Looks good.

File: calibration/TURBDX/CGINS-TURBDX-00993__20240826.csv
Comment (reedan88) [inline calibration/TURBDX/CGINS-TURBDX-00993__20240826.csv:1]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00256__20240926__CC_taarray.ext
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00256__20240926__CC_taarray.ext:1]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240916.csv
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240916.csv:1]: Looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00254__20240916__CC_taarray.ext
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00254__20240916__CC_taarray.ext:1]: Looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00245__20241015__CC_tcarray.ext
Comment (reedan88) [inline calibration/OPTAAD/CGINS-OPTAAD-00245__20241015__CC_tcarray.ext:1]: Looks good.

**Review by reedan88 — APPROVED** [review]
Split review with Collin. No errors found. Looks good.

**Pull request merged**